writing-apps/our-first-app/README.md

```diff
@@ -112,6 +113,8 @@ You can also specify a suggested price in whole USD.
 Remember that AppCenter is a pay-what-you-want store. A suggested price is not a price floor. Users will still be able to choose any price they like, including 0.
 {% endhint %}
 
+The `x-appcenter-stripe` key is used to monetize your app. You can see the value when you trying to submit your app in [AppCenter Dashboard](https://developer.elementary.io/).
```

Comment:
So this is kind of awkward because we don't have this screen in the new AppCenter Dashboard yet. So we need to just provide instructions for existing developers and then maybe throw a bone to new developers. And then since we don't mention the exact key name in the other paragraphs, we can omit it here (as we have a matching key in the example above).

```suggestion
To monetize your app, you also **must include your app's AppCenter Stripe key**. This is a unique public key for each app and is not the same as your Stripe account's regular public key. While the new AppCenter Dashboard is under development, elementary OS 5.1 app developers can find your app's key in its [existing AppStream data](https://appstream.elementary.io/appcenter/html/bionic/main/metainfo/index.html). Developers of new apps will receive their key in the new AppCenter Dashboard once it is live.
```